org.gnome.gitlab.cheywood.Pulp.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "org.gnome.gitlab.cheywood.Pulp",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "command": "pulp",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--share=network",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.secrets"
```

Comment:
This does not work over the portal?

Reply:
I'm not sure? Adding secrets resolved an issue in the past on another project when running on another desktop (non-GNOME).

Are you talking about the others too?

Reply:
> an issue in the past on another project

FYI https://gitlab.gnome.org/World/iotas/-/issues/106

Reply:
That should be resolved from KDE 6 onwards.

Reply:
Ah nvm it's a different issue.